examples/basic_flutter/lib/main.dart

```diff
@@ -40,7 +40,7 @@ class IceCreamPage extends StatefulWidget {
 }
 
 class _IceCreamPageState extends State<IceCreamPage> {
-  var box = Hive['myBox'];
+  var box = Hive.box('myBox');
```

Comment:
hey, could you just help me understanding this?
i think here it should be `Hive.openBox` just like in the other files.
If not, what is different here that it doesn't need to usar de `openBox` method?

Reply:
You are correct, in the `_openBox()` method, `Hive.openBox()` should be called.
Once a box has been opened, it can be accessed without a Future using `Hive.box()`.

This pull request ist WIP and I'll update the documentation to explain the difference in detail...